dotnet-8.0.404.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: dotnet-8.0.404
```

Comment:
This version stream form seems weird to me. Should this be -8.0, or -8? When 8.0.405 comes out, should we delete this?

Reply:
Yeah its semver https://learn.microsoft.com/en-us/dotnet/core/versions/

> - MINOR is incremented when:
>    -  Public API surface area is added.
>    -  A new behavior is added.
>     - A newer MINOR version of an existing dependency is adopted.
>     - A new dependency is introduced.

So 8 or 8.00

Reply:
@xnox given the confusion here can you add more explicit comments in the file itself for the usage and why it is numbered the way it is. So it is apparent to those reading the melange file